Source/Visitable/Visitable.swift

```diff
@@ -17,6 +17,9 @@ public protocol Visitable: AnyObject {
     func visitableDidRender()
     func showVisitableActivityIndicator()
     func hideVisitableActivityIndicator()
+    
+    func visitableDidActivateWebView(_ webView: WKWebView)
+    func visitableDidDeactivateWebView()
```

Comment:
I could provide a default no-op implementation in the protocol extension. This way we don't introduce any breaking changes.

Reply:
I like the idea of a default no-op implementation to avoid breaking changes. Most apps won't have to worry about implementing these for custom behavior anyway.

Reply:
Agree. I've moved them to the protocol extension.